corehq/apps/users/dbaccessors/all_commcare_users.py

```diff
@@ -6,21 +6,26 @@
 
 def get_all_commcare_users_by_domain(domain):
     """Returns all CommCareUsers by domain regardless of their active status"""
-    from corehq.apps.users.models import CommCareUser
+    return imap(CommCareUser.wrap, iter_docs(CommCareUser.get_db(), _get_ids_by_domain(domain)))
+
+
+def get_count_of_commcare_users_in_domain(domain):
+    """Returns count of all CommCareUsers by domain regardless of their active status"""
+    return len(list(_get_ids_by_domain(domain)))
```

Comment:
(And this isn't just nitpicking—many domains have hundreds of mobile works and some have thousands. Pulling all their ids from the database would make the deploy process noticeably slower)